docker/cluster-test/cluster-test.Dockerfile.template

```diff
@@ -17,8 +17,8 @@ RUN docker/cluster-test/compile.sh
 
 FROM debian:buster
 
-RUN apt-get update && apt-get install -y openssh-client curl
-RUN curl -L "https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl" -o /usr/local/bin/kubectl && chmod +x /usr/local/bin/kubectl
+RUN apt-get update && apt-get install -y openssh-client curl wget
+RUN cd /usr/local/bin && wget "https://storage.googleapis.com/kubernetes-release/release/v1.17.0/bin/linux/amd64/kubectl" && chmod +x kubectl
```

Comment:
why fixed version?

Reply:
- know which version was used w/o going through the steps to look up
- triage CT issue
- caching